share/spice/go_watch_it/go_watch_it.js

```diff
@@ -304,26 +300,23 @@
                     quality.push(this.formats[format].quality);
                 }
             }
-            
             if(quality.length > 0) {
                 format_line = "Available in " + quality.join(" / ");
             } else {
                 format_line = "Available";
             }
         }
-        
         return format_line;
     });
-    
+
     Spice.registerHelper("gwi_buyOrRent", function (buy_line, rent_line, options) {
         if (buy_line && buy_line !== "" && !/\$0\.00/.test(buy_line) && /\$/.test(buy_line)) {
```

Comment:
- Can we simplify this to 1 regex with an optional capture?
- We should cache the compiled regex in a variable